packages/manager/apps/hycu/src/hooks/order/useOrderHYCU.ts

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
Add integration test to verify if product catalog is visible and if the order button work well ?

Reply:
That's an e2e test since you are testing other external services

Reply:
You can test `orderLink` corresponding desired value from few params